python/sklearnserver/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sklearnserver"
-version = "0.11.0rc0"
+version = "0.11.0rc1"
```

Comment:
Oh we need to update other places too ? Need to run Andrew’s script I think

Reply:
Done

Reply:
@sivanantha321 : Which script is that ? We would need to make it part of Release process if it is not yet.
cc: @yuzisun

Reply:
it is added to the release process, see step 4
https://github.com/kserve/kserve/blob/master/release/RELEASE_PROCESS_v2.md#on-the-release-day